Cargo.toml

```diff
@@ -7,3 +7,7 @@ edition = "2021"
 crate-type = ["rlib", "cdylib"]
 
 [dependencies]
+dashmap = "5.4.0"
+nix = "0.26.1"
```

Comment:
I'm adding this in my PR - if this is unused, we can leave it out of this PR

Reply:
It's fine if it's added in my opinion, it'll be used in your PR anyway